uqcs/models.py

```diff
@@ -69,6 +70,7 @@ class Student(Member):
     domestic = Column(Boolean)
     year = Column(Integer)
     program = Column(String(100))
+    majors = Column(String(1000))
```

Comment:
Tbh you basically don't have to ever bother providing a length in postgres, `Text` alone is always good enough